### ISSUE TITLE
Rechartering for 2025-26

### ISSUE THREAD

**Issue by svgeesus** [w3c/IFT#248]
Our [existing charter](https://www.w3.org/2023/02/webfonts-2023.html) expires on 28 March 2025. Here is a [draft new charter](https://w3c.github.io/charter-drafts/2025/webfonts-wg.html).

Apart from updating to the latest charter template, and removing Range Request, this is basically the same as the previous charter. The [Motivation and Background](https://w3c.github.io/charter-drafts/2025/webfonts-wg.html#background) is slimmed down a bit, as working on IFT was already approved in the last charter.

[diff to template](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2Fcharter-template.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fwebfonts-wg.html%23background)
[diff to current charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2023%2F02%2Fwebfonts-2023.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2025%2Fwebfonts-wg.html)

**Comment by svgeesus** [comment]
@vlevantovsky @garretrieger

**Comment by vlevantovsky** [comment]
Thank you Chris!

Questions:
1) Will the changes in the "Deliverables" section trigger the need for current participants to re-join the WG and update their IP commitments? (I understand that the changes are trivial, but still - even a trivial change could have an adverse effect if the rechartering process triggers the need for formal step of rejoining the WG.)
2) If the answer to above is "yes" (meaning the rechartering would trigger the need to rejoin), is there a way to avoid it?

Minor nit: the [structure of ISO SC29](https://www.iso.org/committee/45316.html) has changed recently - it used to be comprised of WG1 and WG11 (which effectively was internally structured as the SC), and the former WG11 is now subdivided into multiple WGs. What used to be known as WG11 MPEG Systems Subgroup [responsible for OFF standardization among other projects] is now a standalone SC29/WG3, still responsible for OFF among many other projects.

**Comment by svgeesus** [comment]
> 1. Will the changes in the "Deliverables" section trigger the need for current participants to re-join the WG and update their IP commitments? (I understand that the changes are trivial, but still - even a trivial change could have an adverse effect if the rechartering process triggers the need for formal step of rejoining the WG.)

No. The change is to remove the Range Request spec, which is now discontinued. Reducing scope does not trigger rejoining. (Increasing scope would).

> What used to be known as WG11 MPEG Systems Subgroup [responsible for OFF standardization among other projects] is now a standalone SC29/WG3, still responsible for OFF among many other projects.

Thanks, I will update the draft charter.

**Comment by svgeesus** [comment]
Charter updated for SC29/WG3

**Comment by svgeesus** [comment]
**Announcement:** In terms of the [upcoming 2025 Process](https://www.w3.org/policies/process/drafts/) (which is not yet in effect) we are in the [charter refinement phase](https://www.w3.org/policies/process/drafts/#charter-development) and I am the [chartering facilitator](https://www.w3.org/policies/process/drafts/#chartering-facilitator)

**Comment by garretrieger** [comment]
Updated charter looks good to me.

**Comment by svgeesus** [comment]
Charter has [nearly finished horizontal review](https://github.com/w3c/strategy/issues/486) - just waiting for the privacy review.

**Comment by svgeesus** [comment]
AC review of charter ends this Friday, 19 March.

**Comment by svgeesus** [comment]
Recharter should be announced tomorrow.

**Comment by svgeesus** [comment]
[Rechartered](https://www.w3.org/2025/03/webfonts-wg.html) until 21 March 2027
